tests/baselines/reference/commentsArgumentsOfCallExpression2.js

```diff
@@ -18,6 +18,6 @@ foo(/*c3*/ function () { }, /*d2*/ function () { }, /*e2*/ a + b);
 foo(/*c3*/ function () { }, /*d3*/ function () { }, /*e3*/ (a + b));
 foo(
 /*c4*/ function () { }, 

```

Comment:
@yuit: Anyway, yes, there is a trailing space. I'll have a look if I can fix it. (Tomorrow)